tests/test_kernelIo1.py

```diff
@@ -74,22 +71,9 @@ def testFixedKernel(self):
 
         k = afwMath.FixedKernel(inImage)
 
-        pol = pexPolicy.Policy()
-        additionalData = dafBase.PropertySet()
-        loc = dafPersist.LogicalLocation(
-            os.path.join(testPath, "data", "kernel1.boost"))
-        persistence = dafPersist.Persistence.getPersistence(pol)
-
-        storageList = dafPersist.StorageList()
-        storage = persistence.getPersistStorage("XmlStorage", loc)
-        storageList.append(storage)
-        persistence.persist(k, storageList, additionalData)
-
-        storageList2 = dafPersist.StorageList()
-        storage2 = persistence.getRetrieveStorage("XmlStorage", loc)
-        storageList2.append(storage2)
-        k2 = persistence.unsafeRetrieve(
-            "FixedKernel", storageList2, additionalData)
+        with lsst.utils.tests.getTempFilePath(".fits") as filename:
+            k.writeFits(filename)
+            k2 = afwMath.FixedKernel.readFits(filename)
```

Comment:
I know we're trying to get rid of Boost persistence in general, but why is this part of this ticket?

Reply:
Because these changes broke boost persistence for kernels and I don't know how to fix it (nor do I want to spend any real time on it, since we are not using it and I am about to rip it out).

It might be better to rip it out on this ticket, but I was hoping to keep those tickets separate.